ee/tabby-webserver/src/schema/analytic.rs

```diff
@@ -31,29 +35,58 @@ pub enum Language {
     Other,
 }
 
+lazy_static! {
+    static ref LANGUAGE_STRING_MAPPINGS: HashMap<Language, Vec<&'static str>> = {
```

Comment:
Would rather use a derive or similar for this if possible - it will be very easy to forget to add a value here when a new Language is added

Reply:
switched to a match based clause